bakerydemo/static/css/main.css

```diff
@@ -1048,18 +1509,349 @@ span.outline {
+.picture-card {
+  position: relative;
+}
+
+.picture-card__link:hover .picture-card__image img {
+  transform: scale(1.05);
+}
+
+.picture-card__link:hover .picture-card__title {
+  text-decoration: underline;
+}
+
+.picture-card__title {
+  color: var(--white);
+  font-size: 26px;
+  line-height: 30px;
+}
+
+.picture-card__image {
+  overflow: hidden;
+  margin-bottom: 0;
+}
+
+.picture-card__image img {
+  transition: transform 0.2s ease;
+}
+
+.picture-card__image:before {
+  content: "";
+  position: absolute;
+  height: 200px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, #000000 67.69%);
+  z-index: 2;
+}
+
+.picture-card__contents {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  text-align: left;
+  display: flex;
+  align-items: end;
+  padding: 20px;
+  z-index: 2;
+}
+
+@media (min-width: 766px) {
+  .picture-card__title {
+    font-size: 32px;
+    line-height: 42px;
+  }
+
+  .picture-card__contents {
+    padding: 30px 40px;
+  }
+}
+
+/* #endregion */
+
+/* #region ---- Listing Card ---- */
+.listing-card {
+  margin-bottom: 40px;
+}
+
+.listing-card__meta {
+  border-collapse: separate;
+  border-spacing: 5px;
+}
+
+.listing-card:last-child {
+  margin-bottom: 0;
+}
+
+.listing-card__link {
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 30px;
+  border-bottom: 1px solid var(--border-grey);
+}
+
+.listing-card__contents {
+  padding-top: 10px;
+  padding-left: 25px;
+}
+
+.listing-card__title {
+  color: var(--orange);
+  margin-top: 0;
+}
+
+.listing-card__meta-category {
+  color: var(--grey);
+  font-size: 14px;
+  line-height: 22px;
+  padding-right: 10px;
+  vertical-align: top;
+}
+
+.listing-card__meta-content {
+  color: var(--dark);
+  font-size: 14px;
+  line-height: 22px;
+  vertical-align: top;
+}
+
+.listing-card__meta-row:last-child {
+  margin-bottom: 0;
+}
+
+.listing-card__image {
+  width: 100%;
+  overflow: hidden;
+  margin: 0;
+  max-width: 100px;
+  max-height: 110px;
+}
+
+.listing-card__image img {
+  transition: transform ease 0.2s;
+}
+
+.listing-card:hover .listing-card__image img {
+  transform: scale(1.05);
+}
+
+.listing-card:hover .listing-card__title {
+  color: var(--dark-orange);
+}
+
+@media (min-width: 766px) {
+  .listing-card__contents {
+    padding-left: 45px;
+    padding-top: 30px;
+  }
+
+  .listing-card__image {
+    width: 100%;
+    overflow: hidden;
+    margin: 0;
+    max-width: 180px;
+    max-height: 180px;
+  }
+
+  .listing-card__meta-category,
+  .listing-card__meta-content {
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+}
+
+/* #endregion */
+
+/* #region ---- Blog Listing Card ---- */
+.blog-listing-card {
+  padding-bottom: 40px;
+  width: 100%;
+}
+
+.blog-listing-card:last-child {
+  padding-bottom: 100px;
+}
+
+.blog-listing-card__link {
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 30px;
+  border-bottom: 1px solid rgba(135, 116, 79, 0.25);
```

Comment:
Feels like this color needs to be a CSS variable

Reply:
Added as a transparent border variable